bin/git-stats-importer

```diff
@@ -52,7 +52,7 @@ myRepo.exec("config user.email", function (err, GIT_EMAIL) {
               , all: true
               , "no-color": true
             }, "log");
-            myRepo.exec(command, function (err, data) {
+            myRepo.exec(command + " --pretty=medium --date=default", function (err, data) {
```

Comment:
I tried, but I don't like magic, and couldn't find documentation about that. Couldn't manage to get that to put an equal sign between the arg and the value.